src/core/ext/filters/client_channel/resolver/dns/c_ares/grpc_ares_wrapper_libuv_windows.cc

```diff
@@ -19,65 +19,27 @@
 #include <grpc/support/port_platform.h>
 
 #include "src/core/lib/iomgr/port.h"
-#if GRPC_ARES == 1 && (defined(GRPC_UV) || defined(GPR_WINDOWS))
+#if GRPC_ARES == 1 && defined(GRPC_UV) && defined(GPR_WINDOWS)
```

Comment:
It might make more sense to add `GRPC_RESOLVE_LOCALHOST_MANUALLY` in `iomgr/port.h`, and rename this file to `grpc_ares_wrapper_libuv_localhost.cc`.